examples/inverse/plot_gamma_map_inverse.py

```diff
@@ -64,3 +77,18 @@
 residual.pick_types(meg='grad', exclude='bads')
 residual.plot(titles=dict(grad='Residuals Gradiometers'), ylim=ylim,
               proj=True)
+
+###############################################################################
+# Generate stc from dipoles
+stc = make_sparse_stc_from_dipoles(dipoles, forward)
```

Comment:
I'm also using `forward['source_rr']` to find the index of the source position which corresponds to the dipole location.  `rr` is also in `src`, but for each source space separately, right? So I'd need to build the matrix first by looping of src. Is that better?